package/MDAnalysis/coordinates/DCD.py

```diff
@@ -578,37 +579,39 @@ def timeseries(self, asel=None, start=None, stop=None, step=None, skip=None,
         # XXX needs to be implemented
         return self._read_timeseries(atom_numbers, start, stop, step, format)
 
+    @deprecate(message="This method will be removed in 0.17")
     def correl(self, timeseries, start=None, stop=None, step=None, skip=None):
-        """Populate a :class:`~MDAnalysis.core.Timeseries.TimeseriesCollection` object
-        with time series computed from the trajectory.
-
-        Calling this method will iterate through the whole trajectory and
-        perform the calculations prescribed in `timeseries`.
-
-        Parameters
-        ----------
-        timeseries : :class:`MDAnalysis.core.Timeseries.TimeseriesCollection`
-             The :class:`MDAnalysis.core.Timeseries.TimeseriesCollection` that defines what kind
-             of computations should be performed on the data in this trajectory.
-        start :  int (optional)
-             Begin reading the trajectory at frame index `start` (where 0 is the index
-             of the first frame in the trajectory); the default ``None`` starts
-             at the beginning.
-        stop : int (optional)
-             End reading the trajectory at frame index `stop`-1, i.e, `stop` is excluded.
-             The trajectory is read to the end with the default ``None``.
-        step : int (optional)
-             Step size for reading; the default ``None`` is equivalent to 1 and means to
-             read every frame.
+        """
+Populate a :class:`~MDAnalysis.core.Timeseries.TimeseriesCollection` object
```

Comment:
Could there be a comment about that, then? I am ready to bet it is going to be reverted by accident.